content/04.results.md

```diff
@@ -3,11 +3,66 @@
 ## Random Sampling of Generative Model
 place the grid aurocs here for generative model
 
-## Discriminator Model Builds Off Generative Model
-place the grid of aurocs here for discriminator model
+### Discriminator Model Builds Off Generative Model
+In this framework we used a generative model trained over label functions to produce probabilistic training labels for each sentence.
+Then we train a discriminative model, which has full access to a representation of the text of the sentence, to predict the generated labels.
+The discriminative model is a convolutional neural network trained over word embeddings.
+We report the results of the discriminative model using area under the precision recall curve (AUPR) (Figure {@fig:discriminative_model_auprc}) and area under the receiver operating curve (AUROC) (Figure {@fig:discriminative_model_auroc}).    
```

Comment:
Did you already introduce these earlier in the manuscript? If so, no need to re-define them.